rust/crd/src/lib.rs

```diff
@@ -158,12 +158,23 @@ pub struct HdfsClusterConfig {
 
     /// Settings related to user [authentication](DOCS_BASE_URL_PLACEHOLDER/usage-guide/security).
     pub authentication: Option<AuthenticationConfig>,
+
+    /// Deprecated, please use `.spec.nameNodes.config.listenerClass` and `.spec.dataNodes.config.listenerClass` instead.
```

Comment:
Can you please link to https://github.com/stackabletech/issues/issues/504 for removal?

Reply:
ebdf91649d63604ac2b4d04fda643603d777e78c